Integrations/PhishLabsIOC_DRP/PhishLabsIOC_DRP.py

```diff
@@ -0,0 +1,562 @@
+            List of cases filtered by date
+        """
+        format_time = "%Y-%m-%dT%H:%M:%SZ"
+        begin_date_obj: Optional[datetime] = datetime.strptime(begin_date, format_time) if begin_date else None
+        cases: List = []
+
+        while cases_temp and len(cases) < int(max_records) and last_time:
+            if begin_date_obj:
+                if last_time > begin_date_obj:
+                    cases.append(cases_temp[index])
+                else:
+                    break
+            else:
+                cases.append(cases_temp[index])
+
+            if len(cases) == max_records:
+                break
+            elif len(cases_temp) == index + 1:
+                params['offset'] += len(cases_temp)
+                cases_temp = self._http_request('GET',
+                                                url_suffix=suffix,
+                                                params=assign_params(**params),
+                                                timeout=20).get('data', [])
+                index = 0
+                if not cases_temp:
+                    break
+            else:
+                index += 1
+            last_time = datetime.strptime(cases_temp[index].get(date_field), format_time)
+        return cases
+
+    def get_cases(self, status: Optional[str] = None, case_type: Optional[str] = None,
+                  max_records: Union[str, int] = 20, offset: Union[str, int] = 0,
+                  date_field: str = 'dateModified', begin_date: Optional[str] = None,
+                  end_date: Optional[str] = None, query_type: str = '') -> Dict:
+        """
+        Query the specified kwargs with default parameters if not defined
+
+        Args:
+            status: Filter cases based on the case status
+            case_type: Filter cases by case type
+            max_records: Maximum number of cases to return, default is 20, maximum is 200
+            offset: Paginate results used in conjunction with maxRecords
+            date_field: Field to use to query using dateBegin and dateEnd parameters.
+            begin_date: Date query beginning date
+            end_date: Date query beginning date
+            query_type: query type influence on suffix - all/open/closed
+
+        Returns:
+            Response JSON as dictionary
+        """
+        suffix: str = f'/cases/{query_type}' if query_type else '/cases'
+        params: Dict = {
+            'status': status,
+            'type': case_type,
+            'offset': int(offset),
+            'maxRecords': int(max_records)
+        }
+        raw_response: Dict = self._http_request('GET',
+                                                url_suffix=suffix,
+                                                params=assign_params(**params),
+                                                timeout=20)
+        cases_temp: List = raw_response.get('data', [])
+        # About the drop some mean regex right now disable-secrets-detection-start
```

Comment:
there are no regexes and no secrets here, please remove the comment (same for the end comment)